gcloud/storage/blob.py

```diff
@@ -14,7 +14,7 @@
 
 """Create / interact with Google Cloud Storage blobs."""
 
-import base64
+from base64 import b64encode
```

Comment:
In general, prefer to import namespaces rather than names if possible. Exceptions to this are when unit-testing becomes easier or when a name is long or when a name is exposed as a module-level constant.